website/source/api/index.html.md

```diff
@@ -196,3 +196,14 @@ GET /v1/jobs
 
 Even though these share a path, the `PUT` operation creates a new job whereas
 the `GET` operation reads all jobs.
+
+## HTTP Response Codes
+
+Individual API's will contain further documentation in the case that more
+specific response codes are returned but all clients should handle the following:
+
+* 200 and 204 as success codes.
+* 4xx codes indicate that if a parameter was modified in the request, it could
+  potentially succeed.
+* 403 marks that the client isn't authenticated for the request.
+* 5xx means that the client should not expect the request to succeed if retried.
```

Comment:
This doesn't seem accurate as a 5xx indicates a server error which may be transient. Retries - especially to other servers - _may_ succeed.

In fact I feel like we should encourage retrying to a different server on 5xx responses, but I might be missing something.

Reply:
Yeah this can be a living doc. When we support those codes we can add them